src/main/webapp/src/Routes.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+import ScheduleActivityPage from './pages/ScheduleActivityPage';
+
+import NavigationNavPage from './pages/NavigationNavPage';
+import FormsNavPage from './pages/FormsNavPage';
+import TablesNavPage from './pages/TablesNavPage';
+import AddonsNavPage from './pages/AddonsNavPage';
+import ModalsNavPage from './pages/ModalsNavPage';
+import AdvancedNavPage from './pages/AdvancedNavPage';
+import ComponentsNavPage from './pages/ComponentsNavPage';
+
+// FREE

```

Comment:
Can we only keep the components we need?

Reply:
Yes, I'll add it in the next pull request.